daemon/algod/api/client/restClient.go

```diff
@@ -221,14 +221,20 @@ func (client RestClient) TransactionsByAddr(addr string, first, last, max uint64
 	return
 }
 
+// AssetInformation gets the AssetInformationResponse associated with the passed asset index
+func (client RestClient) AssetInformation(index uint64) (response v1.AssetParams, err error) {
+	err = client.get(&response, fmt.Sprintf("/assets/%d", index), nil)
```

Comment:
The convention is `asset/%d` (no `s` when it's a specific one)

Reply:
fixed, and removed asset by creator as discussed